browser/themes/BUILD.gn

```diff
@@ -2,8 +2,11 @@ source_set("themes") {
   sources = [
     "theme_properties.cc",
     "theme_properties.h",
+    "theme_util.cc",
```

Comment:
these should be in a test only target

Reply:
those apis will be used by settings handler to change theme preference.